test/visibility/rename/renameUsedMod/renameModRenameSymBadAccess2.good

```diff
@@ -1,3 +1,3 @@
 renameModRenameSymBadAccess2.chpl:8: In function 'main':
-renameModRenameSymBadAccess2.chpl:12: error: Symbol 'y' undeclared in module 'Bar'
-note: module 'Bar' was renamed from 'Foo' at renameModRenameSymBadAccess2.chpl:6
+renameModRenameSymBadAccess2.chpl:12: error: cannot find 'y' in module 'Foo'
+renameModRenameSymBadAccess2.chpl:6: note: module 'Foo' was renamed to 'Bar' here
```

Comment:
Pre-existing, but this one seems surprising to me:  'Foo' may not contain 'y', but it obviously contains 'x' which was renamed to 'y'.  What's the point of the `use` statement in this program if not to write expressions exactly like this?  (i.e., I'm guessing we can only refer to `y` in a non-qualified way, but can't make use of `Bar` so why permit both renamings to take place if one of them is pointless?)